core/src/tile/tileManager.cpp

```diff
@@ -415,7 +415,13 @@ void TileManager::loadTiles() {
         auto tileIt = tileSet.tiles.find(tileId);
         auto& entry = tileIt->second;
 
-        if (entry.task && entry.rastersPending() > 0 && !entry.isCanceled()) {
+        // FIXME: When can this happen?
+        if (entry.isCanceled()) {
```

Comment:
I think this can happen because of this: https://github.com/tangrams/tangram-es/pull/715/commits/76c5e007878847342214ff429988fc62de483529#diff-88eb1c828b234b81cac47e5e9c23ab2bR242